### PR TITLE
Add new documentation for maps cluster layer

### PR DESCRIPTION
**Pull request by vagimeli**
### Description
_Describe what this change achieves._
Adds new documentation for maps cluster layer

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._
https://github.com/opensearch-project/documentation-website/issues/3987

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@ananzh Please confirm this feature in on track for 2.11. I'll start drafting documentation in the meantime. Thanks :)

File: _dashboards/visualize/cluster-layer.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- Creating heatmaps
+
+## Prerequisites
+
+To use the maps cluster layer in OpenSearch:
+
+- <what are the prerequistes?>
+- <do we need to reference the Maps Stats API?>
+
+To enable the cluster layer:
```
Comment (vagimeli) [inline _dashboards/visualize/cluster-layer.md:28]: Need testing endpoint.

Comment (vagimeli) [reply _dashboards/visualize/cluster-layer.md:28]: @VijayanB Will you send me the demo and endpoint for testing this feature? I need to wrap up the draft documentation tomorrow. I'll then send you the draft content for your technical review. Thanks!

**Review by vamshin — COMMENTED** [review]
This feature is not being shipped in 2.11. We can skip documentation for now.

**Comment by hdhalter** [comment]
Looks like this moved to 3.0, so I'll update the label.

**Comment by PublisherApplications** [comment]
### Why the merging is blocked?
Merging can be performed automatically with 1 approving review, but it's 1 approving review but why is it not automatically with 1 approving review???

**Comment by PublisherApplications** [comment]
> Looks like this moved to 3.0, so I'll update the label.

yea

**Comment by vagimeli** [comment]
Closing for lack of movement. New PR can be opened when the feature is ready.

**Pull request closed without merging**